No configuration file found.
No pyproject.toml file found.
stubPath /mnt/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/davidsandberg_facenet/af74038a4df2337404736e0d3b80a47d6f928ea8/typings is not a valid directory.
Assuming Python platform Linux
Searching for source files
Found 3 source files
/home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/davidsandberg_facenet/af74038a4df2337404736e0d3b80a47d6f928ea8/a/src/facenet_train.py
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/davidsandberg_facenet/af74038a4df2337404736e0d3b80a47d6f928ea8/a/src/facenet_train.py:39:8 - error: Import "facenet" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/davidsandberg_facenet/af74038a4df2337404736e0d3b80a47d6f928ea8/a/src/facenet_train.py:40:8 - error: Import "lfw" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/davidsandberg_facenet/af74038a4df2337404736e0d3b80a47d6f928ea8/a/src/facenet_train.py:52:12 - error: "makedirs" is not a known member of module (reportGeneralTypeIssues)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/davidsandberg_facenet/af74038a4df2337404736e0d3b80a47d6f928ea8/a/src/facenet_train.py:55:12 - error: "makedirs" is not a known member of module (reportGeneralTypeIssues)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/davidsandberg_facenet/af74038a4df2337404736e0d3b80a47d6f928ea8/a/src/facenet_train.py:138:32 - error: Cannot access member "inference" for type "ModuleType"
    Member "inference" is unknown (reportGeneralTypeIssues)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/davidsandberg_facenet/af74038a4df2337404736e0d3b80a47d6f928ea8/a/src/facenet_train.py:232:18 - error: "xrange" is not defined (reportUndefinedVariable)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/davidsandberg_facenet/af74038a4df2337404736e0d3b80a47d6f928ea8/a/src/facenet_train.py:277:12 - error: "step" is possibly unbound (reportUnboundVariable)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/davidsandberg_facenet/af74038a4df2337404736e0d3b80a47d6f928ea8/a/src/facenet_train.py:203:36 - error: "lfw_paths" is possibly unbound (reportUnboundVariable)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/davidsandberg_facenet/af74038a4df2337404736e0d3b80a47d6f928ea8/a/src/facenet_train.py:204:117 - error: "actual_issame" is possibly unbound (reportUnboundVariable)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/davidsandberg_facenet/af74038a4df2337404736e0d3b80a47d6f928ea8/a/src/facenet_train.py:276:45 - error: "step" is possibly unbound (reportUnboundVariable)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/davidsandberg_facenet/af74038a4df2337404736e0d3b80a47d6f928ea8/a/src/facenet_train.py:294:14 - error: "xrange" is not defined (reportUndefinedVariable)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/davidsandberg_facenet/af74038a4df2337404736e0d3b80a47d6f928ea8/a/src/facenet_train.py:296:18 - error: "xrange" is not defined (reportUndefinedVariable)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/davidsandberg_facenet/af74038a4df2337404736e0d3b80a47d6f928ea8/a/src/facenet_train.py:299:25 - error: "xrange" is not defined (reportUndefinedVariable)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/davidsandberg_facenet/af74038a4df2337404736e0d3b80a47d6f928ea8/a/src/facenet_train.py:364:14 - error: "xrange" is not defined (reportUndefinedVariable)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/davidsandberg_facenet/af74038a4df2337404736e0d3b80a47d6f928ea8/a/src/facenet_train.py:423:28 - error: "learning_rate" is possibly unbound (reportUnboundVariable)
/home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/davidsandberg_facenet/af74038a4df2337404736e0d3b80a47d6f928ea8/a/src/facenet_train_classifier.py
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/davidsandberg_facenet/af74038a4df2337404736e0d3b80a47d6f928ea8/a/src/facenet_train_classifier.py:39:8 - error: Import "facenet" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/davidsandberg_facenet/af74038a4df2337404736e0d3b80a47d6f928ea8/a/src/facenet_train_classifier.py:40:8 - error: Import "lfw" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/davidsandberg_facenet/af74038a4df2337404736e0d3b80a47d6f928ea8/a/src/facenet_train_classifier.py:45:8 - error: Import "h5py" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/davidsandberg_facenet/af74038a4df2337404736e0d3b80a47d6f928ea8/a/src/facenet_train_classifier.py:54:12 - error: "makedirs" is not a known member of module (reportGeneralTypeIssues)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/davidsandberg_facenet/af74038a4df2337404736e0d3b80a47d6f928ea8/a/src/facenet_train_classifier.py:57:12 - error: "makedirs" is not a known member of module (reportGeneralTypeIssues)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/davidsandberg_facenet/af74038a4df2337404736e0d3b80a47d6f928ea8/a/src/facenet_train_classifier.py:167:32 - error: Cannot access member "inference" for type "ModuleType"
    Member "inference" is unknown (reportGeneralTypeIssues)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/davidsandberg_facenet/af74038a4df2337404736e0d3b80a47d6f928ea8/a/src/facenet_train_classifier.py:315:12 - error: "step" is possibly unbound (reportUnboundVariable)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/davidsandberg_facenet/af74038a4df2337404736e0d3b80a47d6f928ea8/a/src/facenet_train_classifier.py:242:50 - error: "lfw_paths" is possibly unbound (reportUnboundVariable)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/davidsandberg_facenet/af74038a4df2337404736e0d3b80a47d6f928ea8/a/src/facenet_train_classifier.py:242:61 - error: "actual_issame" is possibly unbound (reportUnboundVariable)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/davidsandberg_facenet/af74038a4df2337404736e0d3b80a47d6f928ea8/a/src/facenet_train_classifier.py:314:41 - error: "step" is possibly unbound (reportUnboundVariable)
/home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/davidsandberg_facenet/af74038a4df2337404736e0d3b80a47d6f928ea8/a/test/train_test.py
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/davidsandberg_facenet/af74038a4df2337404736e0d3b80a47d6f928ea8/a/test/train_test.py:26:8 - error: Import "cv2" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/davidsandberg_facenet/af74038a4df2337404736e0d3b80a47d6f928ea8/a/test/train_test.py:30:8 - error: Import "facenet_train" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/davidsandberg_facenet/af74038a4df2337404736e0d3b80a47d6f928ea8/a/test/train_test.py:31:8 - error: Import "facenet_train_classifier" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/davidsandberg_facenet/af74038a4df2337404736e0d3b80a47d6f928ea8/a/test/train_test.py:32:8 - error: Import "validate_on_lfw" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/davidsandberg_facenet/af74038a4df2337404736e0d3b80a47d6f928ea8/a/test/train_test.py:33:8 - error: Import "compare" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/davidsandberg_facenet/af74038a4df2337404736e0d3b80a47d6f928ea8/a/test/train_test.py:34:8 - error: Import "visualize" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/davidsandberg_facenet/af74038a4df2337404736e0d3b80a47d6f928ea8/a/test/train_test.py:35:8 - error: Import "test_invariance_on_lfw" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/davidsandberg_facenet/af74038a4df2337404736e0d3b80a47d6f928ea8/a/test/train_test.py:36:8 - error: Import "download_and_extract_model" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/davidsandberg_facenet/af74038a4df2337404736e0d3b80a47d6f928ea8/a/test/train_test.py:41:20 - warning: Class methods should take a "cls" parameter (reportSelfClsParameterName)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/davidsandberg_facenet/af74038a4df2337404736e0d3b80a47d6f928ea8/a/test/train_test.py:53:23 - warning: Class methods should take a "cls" parameter (reportSelfClsParameterName)
33 errors, 2 warnings, 0 infos 
Completed in 2.906sec

time cost(in seconds): 3.0675370693206787
